promptshell/ai_terminal_assistant.py

```diff
@@ -256,7 +258,29 @@ def execute_command(self, user_input: str) -> str:
                 expanded = self.alias_manager.expand_alias(user_input[1:].strip())
                 if expanded != user_input[1:]:
                     print(f"Expanded to: {expanded}")
+                append_history(user_input, expanded)
                 return self.run_direct_command(expanded)
+            if user_input.lower() == "--history":
+                history = get_last_n_history(10)
+                lines = []
+                lines.append("PromptShell Command History")
+                lines.append("\u2500" * 44)
```

Comment:
```suggestion
                lines.append("\u2500" * 44) if platform.system().lower() != "windows" else "-"
```
Windows terminal uses legacy encoding unlike linux that relies on utf-8. Implement the suggested change to avoid encoding issues in windows.